source/robot_model/include/robot_model/Model.hpp

```diff
@@ -206,14 +219,20 @@ inline void Model::set_urdf_path(const std::string& urdf_path) {
   this->urdf_path_->set_value(urdf_path);
 }
 
-inline unsigned int Model::get_nb_joints() const {
+inline unsigned int Model::get_number_of_joints() const {
   // subtract 1 because of the 'universe' joint
```

Comment:
This comment is outdated, maybe say 'number of generalized coordinates'.

Reply:
`get_number_of_joint_frames`? It is becoming a bit verbose

Reply:
no no i was talking about the inline comment on line 223 not about function naming

Reply:
right good catch